.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: bionic
```

Comment:
Shall we migrate to Ubuntu 18.04-LTS (or further 20.04) for all CI hosts?

Reply:
Travis CI does not support Ubuntu 20.04.
Reference: https://docs.travis-ci.com/user/reference/overview/#linux

Currently our Buildkite CI uses Ubuntu 18.04-LTS as well.